### PR TITLE
Change tooltip styles

### PR DESCRIPTION
**Pull request by svetlanamorozko**
### What does this PR do:
Change tooltip styles

### Where should the reviewer start:
Diff

### CODE REVIEW

**Comment by travix-frontend-services** [comment]
Deploy preview for *travix-ui-kit* ready!

Built with commit 3d4683af3213cc9ade02cbc49dc394218f2b84a1

https://deploy-preview-34--travix-ui-kit.netlify.com

**Comment by DzmitryKukharuk** [comment]
One remark:
is this cross button OK for Edge browser ?
image

File: packages/travix-ui-kit/themes/_default.yaml [hunk 1]
```diff
@@ -608,5 +609,5 @@ tooltip:
     shadow: "2px -2px 6px -2px rgba(0,0,0,0.4)"
   shadow: "0 0 6px rgba(0,0,0,0.4)"
   background-color: *blank
-  border-radius: 4px
+  border-radius: 8px
```
Comment (maximuk) [inline packages/travix-ui-kit/themes/_default.yaml]: please use `--tx-generic-border-radius` variable

File: packages/travix-ui-kit/components/tooltip/tooltip.scss [hunk 2]
```diff
@@ -127,25 +136,49 @@
...
 .ui-tooltip__close-button-section {
-  display: flex;
-  flex-direction: row-reverse;
-  margin-bottom: 5px;
+  position: absolute;
+  top: 8px;
+  right: 7px;
+}
+
+.ui-tooltip__close-button {
```
Comment (rbardini) [inline packages/travix-ui-kit/components/tooltip/tooltip.scss]: Remove empty declaration.

File: packages/travix-ui-kit/components/tooltip/tooltip.scss [hunk 3]
```diff
@@ -127,25 +136,49 @@
...
+    background: var(--tx-tooltip-close-button-color);
+    content: '';
+    cursor: pointer;
+    display: block;
+    height: var(--tx-tooltip-close-button-size);
+    position: absolute;
+    right: 50%;
+    top: 0;
+    width: var(--tx-tooltip-close-button-weight);
+  }
```
Comment (rbardini) [inline packages/travix-ui-kit/components/tooltip/tooltip.scss]: Nice work 🙂

Do you know what's the impact of this change on RWD though? I think there are some CSS overrides there that can affect an UI Kit upgrade.

**Comment by rbardini** [comment]
@svetlanamorozko I can't test the tooltip with different positions in the style guide because there are multiple radio inputs with the same ID. Could you please have a look at this? 🙂

**Pull request merged**